<a href="https://colab.research.google.com/github/Robbysaidiii/Scraping_RobbySaidi_mc211d5y2136/blob/main/pelatihan_model_.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## (Tahap pertama)



## Import libarary yang dibutuhkn

In [4]:
import pandas as pd

# Memuat Dataset

In [12]:

from google.colab import drive
drive.mount('/content/drive')
df=pd.read_csv('/content/drive/MyDrive/ulasan_aplikasi_twitter.csv')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


## Exploratory Data Analysis (EDA)

In [14]:
df.head()

userName  Rating                                             Review  \
0  Pengguna Google       2  Banyak bug, jadi lemot (foto atau video sering...   
1  Pengguna Google       1  Saya baru instal lagi x ini, mau login eh "mem...   
2  Pengguna Google       1  semenjak jadi X malah lemot, minta update seri...   
3  Pengguna Google       4  Kok banyak video permasalahan dan lemot padaha...   
4  Pengguna Google       5  Untuk pihak X, mohon tambahkan fitur "juga bis...   

               Tanggal  
0  2025-03-11 15:06:40  
1  2025-03-10 14:18:48  
2  2025-02-04 22:06:28  
3  2025-01-15 13:22:01  
4  2025-03-04 18:53:12

dari dataframe diatas terdapat username, rating,review , tanggal

1.  (username) hanya berisi Pengguna Google yang artinya google menyembunyikan nama penggunna

2.   (Rating) memberika bintang sesuai dengan permasalahan user

3.  (Review) sebuah ekspersi user saat menggunkan aplikasi x/Twitter

4.  (Tanggal) setiap user yag memberika review dan rting akan tercatat tanggalnya



In [15]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 45000 entries, 0 to 44999
Data columns (total 4 columns):
 #   Column    Non-Null Count  Dtype 
---  ------    --------------  ----- 
 0   userName  45000 non-null  object
 1   Rating    45000 non-null  int64 
 2   Review    45000 non-null  object
 3   Tanggal   45000 non-null  object
dtypes: int64(1), object(3)
memory usage: 1.4+ MB


In [16]:
print("Jumlah duplikasi: ", df.duplicated().sum())

Jumlah duplikasi:  0


In [17]:
df.describe(include='all')

userName        Rating           Review              Tanggal
count             45000  45000.000000            45000                45000
unique                1           NaN            44901                44989
top     Pengguna Google           NaN  Sangat membantu  2025-03-10 15:23:42
freq              45000           NaN               12                    2
mean                NaN      2.331089              NaN                  NaN
std                 NaN      1.657138              NaN                  NaN
min                 NaN      1.000000              NaN                  NaN
25%                 NaN      1.000000              NaN                  NaN
50%                 NaN      1.000000              NaN                  NaN
75%                 NaN      4.000000              NaN                  NaN
max                 NaN      5.000000              NaN                  NaN

dari dataframe diatas terdapat 44999 data yang berisi 4 kolom dengan tipe data 3 object dan 1 int64.... ridak ada yang missing value